css/index.css

```diff
@@ -1,3 +1,520 @@
+	margin-top: 50px;
+	margin-left: 35px;
+	height: 310px;
+	width: 60%;
+	text-align: center;
+	line-height: 80px;
+	font-size: 75px;
+}
+
+.main-content .snippy {
+	position: absolute;
+	right: 0;
+	top: 10;
+	height: 380px;
+	width: 40%;
+}
+
+.main-content .start {
+	position: absolute;
+	border: 1px solid gray;
+	left: 230px;
+	bottom: 125px;
+	padding: 5px 45px 3px 45px;
+	height: 35px;
+	font-size: 18px;
+	text-align: center;
+}
+
+/* After what happened ^^^ there, this was pretty simple */
+h2 {
+	font-size: 16px;
+}
+
+.info-content .feature-about {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .feature {
+	padding-top: 10px;
+	margin-right: 20px;
+	width: 45%;
+}
+
+.info-content .about {
+	padding-top: 10px;
+	margin-left: 20px;
+	width: 45%;
+}
+
+.middle-img {
+	margin: 35px 0;
+	width: 100%;
+}
+
+.info-content .ser-pro-vis {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .services {
+	margin-right: 10px;
+	width: 30%;
+	padding-bottom: 25px;
+}
+
+.info-content .product {
+	margin: 0 5px;
+	width: 30%;
+}
+
+.info-content .vision {
+	margin-left: 10px;
+	width: 30%;
+}	
+
+footer {
+	text-align: center;
+	margin-top: 100px;
+}
+
+/* And so begins the tablet break */
+
+@media (max-width: 800px){
+
+    .container {
+        width: 800px;
+    }
+
+    header {
+        height: 130px;
+        width: 800px;
+        position: fixed;
+        top: 0;background-image: none;
+        z-index: 10;
+        display: flex;
+        flex-direction: column;
+    
+    }
+
+    header nav {
+        height: 60px;
+        width: 800px;
+        display: flex;
+        position: relative;
+        line-height: 25px;
+        flex-direction: row;
+        
+    }
+    
+    header nav a {
+        margin: 95px 55px 0 28px;
+        text-decoration: none;
+        justify-content: flex-start;
+        color: rgb(159, 159, 180);
+        line-height: 36px;
+        font-size: 16px;
+        font-weight: bold;
+    }
+
+    
+    header .logo {
+        margin-bottom: 65px;
+        height: 36px;
+        width: auto;
+        align-self: center;
+    }
+    
+    .main-content .snippy {
+        display: none;
+    }
+    
+    .main-content .stack-text {
+        position: absolute;
+        margin-top: 130px;
+        margin-left: 160px;
+        height: 310px;
+        text-align: center;
+        line-height: 73px;
+        font-size: 73px;
+    }
+
+    .main-content .start {
+        position: absolute;
+        border: 1px solid gray;
+        left: 315px;
+        bottom: 67px;
+        padding: 8px 40px 3px 43px;
+        height: 33px;
+        font-size: 17px;
+        text-align: center;
+        font-weight: bold;
+	}
+	
+	.toppy {
+		border-bottom: 2px solid black;
+		margin: 7px 21px 0px 21px;
+	}
+
+	h2 {
+		font-size: 16px;
+	}
+	
+	.info-content .feature-about {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	
+	.info-content .feature {
```

Comment:
info-content feature is great naming for classes, these are very specific and tells you what the contents are which makes it easier to locate on your site.